db/migrate/20200727220143_create_discourse_voting_category_settings.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class CreateDiscourseVotingCategorySettings < ActiveRecord::Migration[6.0]
+  def up
+    create_table :discourse_voting_category_settings do |t|
+      t.integer :category_id
+      t.timestamps
+    end
+    add_index :discourse_voting_category_settings, :category_id, unique: true
+
+    DB.exec <<~SQL
+      INSERT INTO discourse_voting_category_settings(category_id, created_at, updated_at)
+      SELECT category_id, created_at, updated_at
+      FROM category_custom_fields
+      WHERE name = 'enable_topic_voting' and value = 'true'
+    SQL
+
+    DB.exec <<~SQL
```

Comment:
Do we need to delete this in a post migration instead? Otherwise, voting will be disabled before the new code has been deployed? Another suggestion I have is to not delete the custom field first so that if the migration goes wrong, we have a way to reconcile the results.